api/helpers/utils.js

```diff
@@ -223,19 +222,11 @@ exports.runDataQuery = async function (modelType, role, userGuid, query, fields,
                     }
                   ]
                 },
-                // Check if user either has the create-projects role or has project permissions.
-                { $cond: 
-                  { if: { $in: [ "public", role ] }, then: true, else:
```

Comment:
This aggregation query runs for basically all "get" calls, related to the user schema or not. So we need to make sure that "public" records are always returned.